stream_alert_output/main.py

```diff
@@ -120,7 +121,10 @@ def run(self, alerts):
                 output_func = getattr(self, '_{}'.format(output), None)
                 if output_func:
                     self._setup_output_creds(output)
-                    output_func(rule_name, formatted_alerts)
+                    try:
```

Comment:
I intended for it to be a general catchall.  An OutputRequestFailure exception is raised when a failure occurs in request_handler, which wasn't caught by anything.  I can modify this block to only catch OutputRequestFailure, or I can revert it to how it was before.